Sources/Mantis/CropViewController/CropViewController.swift

```diff
@@ -45,13 +51,31 @@ open class CropViewController: UIViewController {
     private var initialLayout = false
     private var disableRotation = false
     
+    private lazy var _undoManager : UndoManager = {
+        return UndoManager()
+    }()
```

Comment:
rename it to `undoManager` to keep consistent naming rule

Reply:
It cannot be renamed undoManager. Every view controller has the default undo manager named "undoManager". This control requires a custom undo manager which is different from the main undo manager that is used to manage the main adjustment stack. This is the proper way to implement this variable.

Reply:
Got it. Thanks for the clarification!